source/index.rst.BAK

```diff
@@ -0,0 +1,20 @@
+.. Fedora Malaysia documentation master file, created by
```

Comment:
Similar to comment for conf.py.BAK. Add this in master first and overwrite it in this branch.